app/config/api_platform/resources.yml

```diff
@@ -0,0 +1,8 @@
+resources:
+  PrestaShop\PrestaShop\Core\Domain\Hook\Query\GetHookStatus:
```

Comment:
I don't think we should use the query as a resource here. Also, returning only `true` or `false` is not optimal IMO. We should probably create a Hook resource that will return something like `{ "active“: true }`.

Reply:
Ok, updated, can you check if It's ok for you ?

Reply:
Why @atomiix, do you think we shouldn't use a query as resource?

Reply:
Because you're not trying to expose the query as a response to the api call, you want a resource/entity

Reply:
I still don't understand but it's not a big deal :)